cinema/models.py

```diff
@@ -0,0 +1,10 @@
+from django.db import models
+
+
+class Movie(models.Model):
+    title = models.CharField(max_length=127)
+    description = models.TextField(blank=True)

```

Comment:
description shouldn't be nullable. Imagine trying to pick a movie and description is none - sounds bad 😸